### ISSUE TITLE
bug: "ImportError: cannot import name 'soft_unicode' from 'markupsafe'" after installing

### ISSUE THREAD

**Issue by mcintyre321** [copier-org/copier#576]
**Describe the problem**

I've just installed Copier for the first time, and when I run any command I get `ImportError: cannot import name 'soft_unicode' from 'markupsafe'`. It seems that a few days ago there was a change in a jinja dependency which has broken things: https://github.com/pallets/markupsafe/issues/282
 
 ```
 E:\>copier
 Traceback (most recent call last):
  File "C:\Program Files\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\x\AppData\Roaming\Python\Python39\Scripts\copier.exe\__main__.py", line 4, in <module>
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\copier\__init__.py", line 3, in <module>
    from .main import *  # noqa
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\copier\main.py", line 18, in <module>
    from .config import make_config
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\copier\config\__init__.py", line 1, in <module>
    from .factory import make_config  # noqa
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\copier\config\factory.py", line 13, in <module>
    from .user_data import load_answersfile_data, load_config_data, query_user_data
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\copier\config\user_data.py", line 11, in <module>
    from jinja2 import UndefinedError
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\jinja2\__init__.py", line 12, in <module>
    from .environment import Environment
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\jinja2\environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\jinja2\defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "C:\Users\x\AppData\Roaming\Python\Python39\site-packages\jinja2\filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (C:\Users\x\AppData\Roaming\Python\Python39\site-packages\markupsafe\__init__.py)
```

**Comment by pawamoy** [comment]
Duplicate of #574

**Comment by Tardo** [comment]
Maybe need fix this version to 2.0.1? https://github.com/copier-org/copier/blob/master/poetry.lock#L294

Required version from jinja for this dependency:
https://github.com/pallets/jinja/blob/df59bde6ae54d2458acce3c25c5d35f38e456a9c/requirements/docs.txt#L24

**Comment by Yajo** [comment]
Find a workaround here: https://github.com/copier-org/copier/issues/574#issuecomment-1046708983

For now, I don't have plans on publishing any more fixes for the 5.x branch, and it is fixed in master (although master has evolved a lot).
